src/main/java/kitchenpos/menu/dto/MenuProductResponse.java

```diff
@@ -0,0 +1,63 @@
+package kitchenpos.menu.dto;
+
+import kitchenpos.menu.domain.MenuProduct;
+
+import java.util.Objects;
+
+public class MenuProductResponse {
+
+    private Long seq;
+    private Long menuId;
+    private Long productId;
+    private long quantity;
+
+    public MenuProductResponse() {
+    }
+
+    public MenuProductResponse(Long seq, Long menuId, Long productId, long quantity) {
+        this.seq = seq;
+        this.menuId = menuId;
+        this.productId = productId;
+        this.quantity = quantity;
+    }
+
+    public static MenuProductResponse of(MenuProduct menuProduct) {
+        return new MenuProductResponse(
+                menuProduct.getSeq()
+                , menuProduct.getMenu().getId()
+                , menuProduct.getProduct().getId()
+                , menuProduct.getQuantity());
```

Comment:
사소하지만 아래와 같이 라인을 구분해보는건 어떨까요 ?

```suggestion
                menuProduct.getSeq(), 
                menuProduct.getMenu().getId(),
                menuProduct.getProduct().getId(),
                menuProduct.getQuantity());
```

Reply:
현재 라인 구분 컨벤션이 전부 앞 콤마로 되어있어서 이 부분은 유지하겠습니다! ㅎㅎ